### PR TITLE
Accordion: @emotion, isOpen fix, percy test update

### PR DESCRIPTION
**Pull request by TrygveUrdahl**
# PR Checklist
https://elvia.atlassian.net/wiki/spaces/TEAMATOM/pages/10427498683/Review+prosess

- [ ] Bumpet package?
- [ ] Updated changelog?
- [ ] Correct date in changelog?

### CODE REVIEW

File: packages/components/components/elvis-accordion/src/react/elvia-accordion.spec.tsx [hunk 1]
```diff
@@ -4,20 +4,109 @@ import React from 'react';
...
+            tenetur temporibus alias aliquam eum praesentium nisi? Repudiandae dolor perspiciatis est sapiente
+            porro.
+          </div>
+        }
+      />
+    </div>,
+  );
+  await page.getByRole('button', { name: 'Click to open accordion', exact: true }).click();
+  await page.waitForTimeout(300);
+  await percySnapshot(page, 'Accordions', { widths: [900] });
```
Comment (TrygveUrdahl) [inline packages/components/components/elvis-accordion/src/react/elvia-accordion.spec.tsx:111]: There is no point in running these tests for multiple widths as the component doesn't change for desktop vs. mobile, so I'm here explicitly only running it for one screen width to reduce the number of Percy screenshots

Comment (TomSchrier) [reply packages/components/components/elvis-accordion/src/react/elvia-accordion.spec.tsx:111]: Does the typography automatically change between mobile and desktop?

**Review by eTallang — COMMENTED** [review]
LGTM when Toms comment is resolved 🙂

File: packages/components/components/elvis-accordion/src/react/elvia-accordion.spec.tsx (hunk 1 above)
Comment (TrygveUrdahl) [reply packages/components/components/elvis-accordion/src/react/elvia-accordion.spec.tsx:111]: Generally no, however a user can override the typography used for the buttons and that could give it some changes. 
However, I would argue that such a minor change would fall outside the scope of these tests, and should instead be covered by typography-specific tests (which I believe we have in the Elvis percy-tests).

**Pull request merged**